python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml

```diff
@@ -0,0 +1,23 @@
+name: dining_prefs
+file_type: .csv
+author: J.L. Moreno
```

Comment:
Between `file_type` and `author`, we should add a `description` field that briefly explains what this graph represents